examples/integrations/langchain4j/coffee-shop-assistant-mp/src/main/java/io/helidon/examples/integrations/langchain4j/mp/coffee/shop/assistant/ai/MenuItemsIngestor.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.examples.integrations.langchain4j.mp.coffee.shop.assistant.ai;
+
+import java.util.logging.Logger;
+
+import io.helidon.examples.integrations.langchain4j.mp.coffee.shop.assistant.data.MenuItem;
+import io.helidon.examples.integrations.langchain4j.mp.coffee.shop.assistant.data.MenuItemsService;
+
+import dev.langchain4j.data.document.Document;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.model.embedding.onnx.allminilml6v2.AllMiniLmL6V2EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIngestor;
+import dev.langchain4j.store.embedding.inmemory.InMemoryEmbeddingStore;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.Initialized;
+import jakarta.enterprise.event.Observes;
+import jakarta.enterprise.inject.Produces;
+import jakarta.inject.Inject;
+
+@ApplicationScoped
+public class MenuItemsIngestor {
+    private static final Logger LOGGER = Logger.getLogger(MenuItemsIngestor.class.getName());
+
+    private final MenuItemsService menuItemsService;
+
+    @Produces
+    private final EmbeddingModel embeddingModel = new AllMiniLmL6V2EmbeddingModel();
```

Comment:
Please do not use `private` on fields/methods that are supposed to be used by CDI. 
Even though CDI can handle it, it breaks encapsulation.